packer/install-dependencies.sh

```diff
@@ -370,14 +370,17 @@ echo "  - busybox" >> ${RELEASE_NOTES_FILEPATH}
 # TODO: fetch supported k8s versions from an aks-engine command instead of hardcoding them here
 K8S_VERSIONS="
 1.15.2
+1.15.2-azs
 1.15.1
 1.15.1-azs
 1.14.5
+1.14.5-azs
 1.14.4
 1.14.4-azs
 1.14.3
```

Comment:
can you please remove 1.14.3 and 1.13.7?

Reply:
(both the azure stack and the public cloud version)

Reply:
Ah yes, thx @CecileRobertMichon 

We want to reduce any unused hyperkube image versions onto the VHD.